### DISCUSSION TITLE
求助，我的邦布失踪了。

### DISCUSSION THREAD

**Discussion by imhlandid** [share121/inter-knot#2528] [其他]
![358566080-536e6c76-7703-4888-a12d-1a8e662c4aa9](https://github.com/user-attachments/assets/6b7ad303-aa8b-4293-89ea-4acb1793cbec)
【委托】 我的朋友名字是泰迪，在光映广场的引力电影院失踪了，说来也怪我我和它看电影的时候，我就想吃爆米花它就去给我买，结果失踪了。泰迪的耳朵有为了保护我导致磕到的小窝窝，泰迪的小屁屁有一个被我弄上去的胶水很粘则么也弄不下来。它高101cm重32千克希望有能力的绳匠可以帮我找回来。我爸爸不让我去治安局所以只能来绳网了QAQ

**Comment by huiye123-Hub** [comment]
这个委托我接不了<img src="https://bbs-static.miyoushe.com/static/2023/12/19/53a316bdef4a16e29dc8d5bb8a0be976_1706072813871686262.png" alt="螺丝咕姆 宕机" width="75" >

**Comment by ThReeCj** [comment]
這委托我接了

**Comment by luan939150334** [comment]
说真的，我本以为帮布只有30到60CM高，结果1M多，光映广场充电的帮布宽才两个鞋子大，治安布算上耳朵最多三个鞋子高，假设一个鞋子长23CM这才69CM。你一个1米不到的孩子和一个1米左右的帮布玩也太吓人了吧。

**Comment by wer-456** [comment]
这个委托我接了

**Comment by chenx198** [comment]
这个委托我完成了 ![image](https://github.com/user-attachments/assets/11bb9f57-0e2d-43fa-88a0-76b479f8a5fe)

**Comment by opqphp** [comment]
你的邦布被用我十发邦布券拐走了<img src="https://img-static.mihoyo.com/communityweb/upload/546238a8f5fc2ab5b36f5aae5afcfa16.png" alt="胡桃-爱心" width="75" >

**Comment by 56712354** [comment]
这个委托我不接。。。
